packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-## NEXT
+## 3.0.0
 
+* **BREAKING CHANGE**: On Android, hybrid composition (SurfaceAndroidWebView)
```

Comment:
I need to think about this one, but at a high level, I don't see why changing the backend should be considered a breaking change.

Reply:
The default behavior is fundamentally changing; it's safer to do it as part of a breaking change.